test/e2e_node/topology_manager_test.go

```diff
@@ -520,7 +571,107 @@ func teardownSRIOVConfigOrFail(f *framework.Framework, sd *sriovData) {
 	framework.ExpectNoError(err)
 }
 
-func runTopologyManagerNodeAlignmentSuiteTests(f *framework.Framework, configMap *v1.ConfigMap, reservedSystemCPUs string, numaNodes, coreCount int, policy string) {
+func runTMScopeResourceAlignmentTestSuite(f *framework.Framework, configMap *v1.ConfigMap, reservedSystemCPUs, policy string, numaNodes, coreCount int) {
+	threadsPerCore := 1
+	if isHTEnabled() {
+		threadsPerCore = 2
+	}
+	sd := setupSRIOVConfigOrFail(f, configMap)
+	var ctnAttrs, initCtnAttrs []tmCtnAttribute
+
+	envInfo := &testEnvInfo{
+		numaNodes:         numaNodes,
+		sriovResourceName: sd.resourceName,
+		policy:            policy,
+		scope:             podScopeTopology,
+	}
+
+	ginkgo.By(fmt.Sprintf("Admit two guaranteed pods. Both consist of 2 containers, each container with 1 CPU core. Use 1 %s device.", sd.resourceName))
+	ctnAttrs = []tmCtnAttribute{
+		{
+			ctnName:       "ps-container-0",
+			cpuRequest:    "1000m",
+			cpuLimit:      "1000m",
+			deviceName:    sd.resourceName,
+			deviceRequest: "1",
+			deviceLimit:   "1",
+		},
+		{
+			ctnName:       "ps-container-1",
+			cpuRequest:    "1000m",
+			cpuLimit:      "1000m",
+			deviceName:    sd.resourceName,
+			deviceRequest: "1",
+			deviceLimit:   "1",
+		},
+	}
+	runTopologyManagerPositiveTest(f, 2, ctnAttrs, initCtnAttrs, envInfo)
+
+	numCores := threadsPerCore * coreCount
+	coresReq := fmt.Sprintf("%dm", (numCores/2+1)*1000)
+	ginkgo.By(fmt.Sprintf("Admit a guaranteed pod requesting %d CPU cores, i.e., more than can be provided at every single NUMA node. Therefore, the request should be rejected.", numCores+2))
+	ctnAttrs = []tmCtnAttribute{
+		{
+			ctnName:       "gu-container-1",
+			cpuRequest:    coresReq,
+			cpuLimit:      coresReq,
+			deviceRequest: "1",
+			deviceLimit:   "1",
+		},
+		{
+			ctnName:       "gu-container-2",
+			cpuRequest:    coresReq,
+			cpuLimit:      coresReq,
+			deviceRequest: "1",
+			deviceLimit:   "1",
+		},
+	}
+	runTopologyManagerNegativeTest(f, ctnAttrs, initCtnAttrs, envInfo)
+
+	// The Topology Manager with pod scope should calculate how many CPUs it needs to admit a pod basing on two requests:
+	// the maximum of init containers' demand for CPU and sum of app containers' requests for CPU.
+	// The Topology Manager should use higher value of these. Therefore, both pods from below test case should get number of CPUs
+	// requested by init-container of highest demand for it. In result both pods should be placed on distinct NUMA nodes.
+	coresReq = fmt.Sprintf("%dm", (numCores/2+1)*1000)
```

Comment:
I wonder if a simple constant > 2 could be clearer here. Not a big deal.

Reply:
You are right, it would be enough, but this TC checks correct assignment of resources for 2 pods also.
We want these two to get CPUs from distinct NUMA nodes, since coresReq is slightly higher than a half of available resources on each node.

Reply:
Fair enough for me: it's a good reason. Please make sure to reflect this answer in the above comment, so your reasoning is documented.

Reply:
Thanks for your opinion on that - I added some words to clarify this TC to make it more informative.